src/matrix/sessioninfo/localstorage/SessionInfoStorage.ts

```diff
@@ -22,6 +22,7 @@ interface ISessionInfo {
     homeServer: string; // deprecate this over time
     accessToken: string;
     lastUsed: number;
+    isGuest: boolean;
```

Comment:
@psrpinto One alternative that I realised after doing this, is we can avoid touching this interface if we utilise the `whoami` endpoint to set the flag on session. Thoughts?

Reply:
Makes sense!